src/components/Modal/Modal.tsx

```diff
@@ -53,6 +51,9 @@ export function Modal({
     'aria-label': ariaLabel,
     container,
     qa,
+    enableFocusTrap,
+    focusTrapOptions,
+    focusTrapRef,
```

Comment:
I don't think we should give direct access to the `focus-trap` options, we need to use some abstraction here.